src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeption.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private static final RegistrationService registrationService = new RegistrationServiceImpl();
+    private static final Integer DEFAULT_AGE = 18;
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final String DEFAULT_LOGIN = "Ukraine";
+    private User defaultUser;
+
+    @BeforeEach
+    void setUp() {
+        defaultUser = new User();
+        defaultUser.setAge(DEFAULT_AGE);
+        defaultUser.setPassword(DEFAULT_PASSWORD);
+        defaultUser.setLogin(DEFAULT_LOGIN);
+    }
+
+    @AfterEach
+    void clearAfterTest() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_NullAge_NotOk() {
+        User actual = defaultUser;
+        actual.setAge(null);
```

Comment:
```suggestion
        defaultUser.setAge(null);
```

Reply:
same for other cases, don't need to create new object if you already have default user